renderer/native/android/src/main/java/com/tencent/renderer/component/text/FontLoader.java

```diff
@@ -0,0 +1,157 @@
+/* Tencent is pleased to support the open source community by making Hippy available.
+ * Copyright (C) 2018 THL A29 Limited, a Tencent company. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.tencent.renderer.component.text;
+
+import android.text.TextUtils;
+
+import androidx.annotation.NonNull;
+
+import com.tencent.mtt.hippy.modules.Promise;
+import com.tencent.mtt.hippy.utils.ContextHolder;
+import com.tencent.mtt.hippy.utils.LogUtils;
+import com.tencent.renderer.NativeRender;
+import com.tencent.vfs.ResourceDataHolder;
+import com.tencent.vfs.VfsManager;
+import com.tencent.vfs.VfsManager.FetchResourceCallback;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class FontLoader {
+
+    private final VfsManager mVfsManager;
+    private final File mFontDir;
+    private final File mFontUrlDictPath;
+    private HashMap<String, String> mFontUrlDict;
+    private static final String[] allowedExtensions = {"otf", "ttf"};
```

Comment:
Rename allowedExtensions to ALLOWED_EXTENSIONS in all caps for better clarity and adherence to Java naming conventions for constants